recipe/meta.yaml

```diff
@@ -19,13 +19,15 @@ source:
   sha256: {{ hash }}
 
 build:
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 1
+  script: {{ PYTHON }} -m pip install . --no-deps -vv  # [not unix]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv && echo complete -C aws_complete aws >aws && ginstall -D aws $PREFIX/share/bash-completions/completions/aws  # [unix]
```

Comment:
is this not a file that already exists in the awscli sources? seems weird to create it in the build location then use ginstall vs just writing it straight into the prefix, if it's not part of the source

Reply:
No, it doesn't exist in the awscli sources.  They document how one can [Enable command completion](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-completion.html) but they don't do it as a part of their pip install.  Given the variety of flavors of completion that can use the aws_completer, I can understand why awscli doesn't include this file.  However, there are examples of system packages that do this so that aws_completer is used if bash-completion is also installed.

I was creating it and using install because it seemed shorter than doing `mkdir -p $PREFIX/share/bash-completions/completions` and then writing it straight into the file.  I'm happy to change it around to be longer if you feel like it is more intuitive.  At least BSD `mkdir` supports `-p` and we wouldn't have to add `gnu-coreutils`.

Reply:
the latter feels slightly more readable / intuitive to me and avoids creating (potentially overwriting even) an unused temp file. it's been a while since I wrote anything that used gnu install so the `-D` flag isn't one I've committed to memory

Reply:
also is the content of the created file missing the r in `aws_completer` here?

Reply:
Yes.  Thanks for catching the missing `r`.